src/service.js

```diff
@@ -32,6 +33,38 @@ import {dev} from './log';
 let ServiceHolderDef;
 
 /**
+ * Returns a service with the given id. Assumes that it has been constructed
+ * already.
+ * @param {!Window} win
+ * @return {!Object} The service.
+ */
+export function getExistingServiceForWindow(win, id) {
+  if (getMode().localDev) {
+    dev.assert(
+       win.services && win.services[id] && win.services[id].obj,
```

Comment:
Well, with that one could kill the entire assert but it produces a much nicer error. Should all be optimized away.